docs/conf/portland/2024/writing-day.rst

```diff
@@ -151,6 +153,17 @@ It gets better! For every challenge completed, you'll receive one (1) entry for
 
 Already familiar with GitBook? We're looking for people like you to brainstorm features as. Help us make the ultimate documentation platform!
 
+Use OpenAPI with the Museum API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Project organizer Heather Cloward, she/her.
+
+The `Museum API <https://github.com/Redocly/museum-openapi-example/tree/main>`__is a sample OpenAPI description. It offers a set of endpoints to interact with a museum's services -- such as retrieving museum hours, managing special events, and purchasing tickets.
+
+Our Writing Day goal is to onboard contributers and improve the Museum API definition. Help us create more complex use case examples like discriminator, x-webhooks, nullable, recursive property, oneOf example, etc. 
+
+Our end goal is to update the Museum API to better demonstrate all that OpenAPI has to offer.
```

Comment:
```suggestion
The Museum API was built specifically to teach OpenAPI concepts and to create a better developer experience. The more realistic and complex the Museum API is, the better it can demonstrate all that OpenAPI has to offer.
```